WebDriverAgentLib/Categories/XCUIElement+FBTVFocuse.m

```diff
@@ -24,9 +24,9 @@ @implementation XCUIElement (FBTVFocuse)
 
 - (BOOL)fb_setFocusWithError:(NSError**) error
 {
-  [[FBApplication fb_activeApplication] fb_waitUntilSnapshotIsStable];
+  [FBApplication.fb_activeApplication fb_waitUntilSnapshotIsStable];
 
-  if (!self.wdEnabled) {
+  if (!self.isEnabled) {
```

Comment:
why not wdEnabled?

Reply:
When I do `po self.wdEnabled` on debugging, no property error raised in XCUIElement. So just changed to `isEnabled`. (backed to wdEnabled. It also worked)